desktop/components/credit_card/client/error_handling_form.coffee

```diff
@@ -58,6 +59,7 @@ module.exports = class ErrorHandlingForm extends Backbone.View
       # we either want to override or fallback to the message sent from gravity
       if errorJson.error?
         message = @errors[errorJson.error] || errorJson.error
+        message = message() if ( _.isFunction message)
```

Comment:
Ah, yes- I started with this inner function because I was having Force re-check the model's sale status for whether it was closed or live. Then I decided to keep all the logic in gravity since causality will do this in the future... Thank you sir!